src/components/challenge-result-dialog.tsx

```diff
@@ -1,111 +1,77 @@
-"use client";
+'use client'
```

Comment:
Updated this file to use the `challengeLink` and `githubRepo` results returned from `/generate-challenge`